mindbender/plugins/validate_pipeline_filesaved.py

```diff
@@ -11,7 +11,7 @@ class ValidateCurrentSaveFile(pyblish.api.ContextPlugin):
     """
 
     label = "Saved file"
-    order = pyblish.api.CollectorOrder + 0.1
+    order = pyblish.api.ValidatorOrder - 0.1
```

Comment:
it is not super important but I though it best that it is the first thing to show the user that they have been careless and not saved.